README.betzy

```diff
@@ -1,38 +1,21 @@
 !+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 1) Recommended .bashrc content for BETZY
 
-ml purge
-module load Python/3.8.2-GCCcore-9.3.0
-module load ESMF/8.0.1-intel-2020a
-module load FFTW/3.3.8-intel-2020a
-module load UDUNITS/2.2.26-GCCcore-9.3.0
-module load CMake/3.16.4-GCCcore-9.3.0
-module load intel/2021a
-
-PS. If using OpenMPI software, It is recommended to use the following modules :
-
 ml purge
 ml load UDUNITS/2.2.26-GCCcore-9.3.0
 ml load CMake/3.16.4-GCCcore-9.3.0
 ml load Python/3.8.2-GCCcore-9.3.0
 ml load FFTW/3.3.8-iomkl-2020a
 ml load ESMF/8.0.1-iomkl-2020a
 
+ulimit -s 2000000   ## if needed
 
 
 2) Compiling hycom_all under NERSC-HYCOM-CICE/hycom/hycom_ALL/hycom_2.2.72_ALL
 
    # Define "setenv ARCH intelIFC" in Make_all.src
    csh Make_all.com
 
-   !! NOT COMPLETE
-	The csh script above does not complete because it gets stuck in archive folder.
-        Normally it should fail and continue but not this time.
-        You can compile each folder individually for the time being until the problem is resolved.
-
-        Netcdf routines are not tested and are not vital for HYCOM, but for those interested,
```

Comment:
I would keep the part about the NetCDF routines in case someone wants to compile them

Reply:
Done